app/templates/index.hbs

```diff
@@ -13,7 +13,8 @@
 
     </div>
   </div>
-  <h1  class="font-weight-light mt-4 text-center m-4">Agencies</h1>
+  <h1  class="font-weight-light mt-4 text-center m-4">PASS supports these agencies' and institutions'<br> public and open access policies</h1>
```

Comment:
Maybe here have `Supported Agencies and Institutions` as the header, and then have a h2/h3 or p tag saying the rest?

Reply:
Also, unrelated, but maybe add a "?" to the Interested h1 element?

Reply:
i agree with cam's suggestion

Reply:
Looks good pending tests!